config/packages/doctrine.yaml

```diff
@@ -11,15 +11,39 @@ doctrine:
 
         # IMPORTANT: You MUST configure your server version,
         # either here or in the DATABASE_URL env var (see .env file)
-        server_version: '5.7'
+        #server_version: '13'
```

Comment:
The reason I touched it at all, was that the Symfony Flex recipe for doctrine was updated, so when I updated the recipes, this was changed. Since we don't use this value (it's set in the database DSN in the env files), I choose not to touch this manually.

Reply:
still not completely happy with this. but I understand the reasoning. So lets leave it as is